fronts-client/src/bundles/editionsIssueBundle.ts

```diff
@@ -17,3 +22,47 @@ export const {
 export type EditionsIssueState = State<EditionsIssue>;
 
 export default reducer;
+
+export const getEditionIssue = (
+  id: string
+): ThunkResult<Promise<void>> => async (dispatch: Dispatch) => {
+  try {
+    dispatch(actions.fetchStart());
+    const issuePromise = getIssueSummary(id);
+    const version = await getLastProofedIssueVersion(id);
+    const i = {
+      ...(await issuePromise),
+      version,
+    };
+    dispatch(actions.fetchSuccess(i));
+  } catch (error) {
+    dispatch(actions.fetchError('Failed to get issue'));
+  }
+};
+
+export const refreshEditionVersion = (
+  issueId: string
+): ThunkResult<Promise<void>> => async (dispatch, getState) => {
+  try {
+    dispatch(actions.fetchStart());
+    // Get current issue
+    const issue = selectors.selectAll(getState());
+    if (!issue) {
+      // wtf??
```

Comment:
Do we know why/how this can happen?

Reply:
It can't, but you need it to help typescript be sure that the pathway has been explored, so you can get the value after.